addons/l10n_be_pos_sale/tests/test_l10n_be_pos_sale.py

```diff
@@ -29,7 +29,7 @@ def test_settle_order_is_invoice(self):
             'order_line': [Command.create({
                 'product_id': self.product_a.id,
                 'product_uom_qty': 10,
-                'product_uom': self.product_a.uom_id.id,
+                'product_uom_id': self.product_a.uom_id.id,
```

Comment:
FYI: whenever the `product_uom_id` is set to the uom of the product given as `product_id` in the create values, you might as well drop the value, as it'll be computed and assigned the same value in `_compute_product_uom_id`.
You don't have to, but it could simplify the code and remove some 'useless' lines.